File: 프로그래머스/1/77484. 로또의 최고 순위와 최저 순위/로또의 최고 순위와 최저 순위.js
function solution(lottos, win_nums) {
    let 당첨번호수 = 0;
    let 동생놈이가린번호개수 = lottos.filter(lotto=>lotto===0).length;
   
    lottos.forEach(lotto=> win_nums.includes(lotto) && 당첨번호수++)
    let 최소등수 = 7-당첨번호수 === 7 ? 6: 7-당첨번호수;
    let 최대등수 = 7-(당첨번호수+동생놈이가린번호개수) === 7 ? 6 : 7-(당첨번호수+동생놈이가린번호개수);
    
    return [최대등수, 최소등수]
}

File: 프로그래머스/1/77484. 로또의 최고 순위와 최저 순위/로또의 최고 순위와 최저 순위.py
def solution(lottos, win_nums):
    winCount = 0;
    zeroCount = 0;
    for w in win_nums:
        if w in lottos:
            winCount += 1
    
    for l in lottos:
        if l == 0:
            zeroCount += 1
    
    최소등수 = 6 if winCount <= 1 else 7 - winCount;
    최대등수 = 6 if zeroCount + winCount <= 1 else 7 - zeroCount - winCount
    

    
    
    return [최대등수, 최소등수]